pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
```

Comment:
OK, if you do need CMake flags (as opposed to just passing `null` instead of some buildInputs) and there are also documentation/examples flags, then this makes more sense.

Your PR definitely looked like you wanted to use the expression as-is, if you have immediate plans to update the expression, comment on the PR after pushing everything (GitHub notifications about force-pushes to PRs work in some weird way).